### PR TITLE
[Tutorial] Add Paymaster + Wagmi tutorial and remove large .gif for OCK theme tutorial

### PR DESCRIPTION
**Pull request by hughescoin**
**What changed? Why?**

- Add new tutorial for Paymaster integration for Wagmi apps
- Update links in OCK Fund tutorial
- Remove large .gif file from OCK Theme tutorial

**Notes to reviewers**

**How has it been tested?**
yes

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

File: apps/base-docs/tutorials/docs/1_ock-swap-theme.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+slug: /create-custom-themed-swap-component
+description: Learn how to implement a swap component with a custom theme using OnchainKit in your React application.
+author: hughescoin
+keywords: [OnchainKit, Swap Component, Custom Theme, React, TypeScript, ERC20 Tokens, Base Chain]
+tags: ['frontend', 'defi', 'ethereum', 'base']
+difficulty: medium
+displayed_sidebar: null
+---
+
+In this tutorial, you'll learn how to create a swap component with a custom theme using OnchainKit. We'll start with the OnchainKit App Template and modify it to include a swap interface for ERC20 tokens on the Base.
```
Comment (ericbrown99) [inline apps/base-docs/tutorials/docs/1_ock-swap-theme.md:12]: "on the Base" -> "on Base"

File: apps/base-docs/tutorials/docs/1_ock-swap-theme.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+
+---
+
+## Objectives
+
+By the end of this tutorial, you should be able to:
```
Comment (ericbrown99) [inline apps/base-docs/tutorials/docs/1_ock-swap-theme.md:18]: "you should be" ->  "you will be"

File: apps/base-docs/tutorials/docs/1_ock-swap-theme.md [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+
+## Prerequisites
+
+### React and TypeScript
+
+You should be familiar with React and TypeScript. If you're new to these technologies, consider reviewing their [official documentation](https://react.dev/) first.
+
+### OnchainKit
+
+This tutorial uses OnchainKit. Familiarity with its basic concepts will be helpful.
```
Comment (ericbrown99) [inline apps/base-docs/tutorials/docs/1_ock-swap-theme.md:33]: Link to OnchainKit for ease

**Review by ericbrown99 — COMMENTED** [review]
Noted a couple typos, otherwise looks good!

**Review by wbnns — CHANGES_REQUESTED** [review]
@hughescoin 

Heya, looks like this still brings in the large gif; can you please remove?

2024-11-02 at 13 52 51@2x

**Review by ericbrown99 — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for ericbrown99 @ 2024-11-04 18:37:55 UTC
User failed mfa authentication, see [go/mfa-help](http://go/mfa-help)

**Pull request closed without merging**